salt/loader.py

```diff
@@ -1,1685 +1,35 @@
 # -*- coding: utf-8 -*-
```

Comment:
``salt/loader.py`` was gutted with core functions moving to ``salt/loader_core.py``